qiskit/providers/models/backendconfiguration.py

```diff
@@ -345,10 +341,32 @@ def __init__(self,
                          n_uchannels=n_uchannels, u_channel_lo=u_channel_lo,
                          meas_levels=meas_levels, qubit_lo_range=qubit_lo_range,
                          meas_lo_range=meas_lo_range,
-                         dt=dt * 1e-9, dtm=dtm * 1e-9,
+                         _dt=dt * 1e-9, dtm=dtm * 1e-9,
```

Comment:
I don't understand why `dt` is changed to `_dt` but `dtm` is left the same. Should both be changed or remain the same (I'm thinking they should both be without underscores).

Reply:
I don't think either are necessary